src/pages/Attendance.jsx

```diff
@@ -1,11 +1,98 @@
+import React from "react";
+import dayjs from "dayjs";
+import { Stack, styled } from "@mui/material";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import ChurchIcon from "@mui/icons-material/Church";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+import SettingsIcon from "@mui/icons-material/Settings";
 import Layout from "../components/Layout/Layout";
+import Title from "../components/shared/Title";
 
 const Attendance = () => {
+  const [value, setValue] = React.useState(new Date());
+  const [openCalendar, setOpenCalendar] = React.useState(false);
+
+  const handleChange = (newData) => {
+    setValue(newData);
+    setOpenCalendar(false);
+  };
+
+  const settings = {
+    MuiPickersToolbar: {
+      styleOverrides: {
+        root: {
+          backgroundColor: "#000",
+        },
+      },
+    },
+  };
+  const handleCalendarClick = () => {
+    setOpenCalendar(true);
+  };
   return (
     <Layout>
-      <div>Attendance</div>
+      <AttendanceWapper>
```

Comment:
spell Checker 익스텐션 깔아. 오타가 많음
```suggestion
      <AttendanceWrapper>
```